be/src/runtime/file_result_writer.h

```diff
@@ -58,6 +58,10 @@ struct ResultFileOptions {
     int write_buffer_size_kb;
     THdfsProperties hdfs_properties;
     bool use_broker;
+    int64_t max_row_group_bytes{0};
```

Comment:
extract a new struct to encapsulates parquet related options

Reply:
done